### ISSUE TITLE
"key invalid" if incorrect regions are selected although the key is actually valid. 

### ISSUE THREAD

**Issue by iceq2** [MicrosoftTranslator/DocumentTranslator#134]

**Comment by chriswendt1** [comment]
I agree. To conclude that the key is invalid in all regions would require to test it in all regions. 
Maybe we can change the mesage to say that the key is invalid in the selected region and suggest to choose a different region.

**Comment by iceq2** [comment]
I would rather put it like:
Please make sure the key was copied correctly and that the correct region
was selected in both fields.
(Unless the system knows for sure when the error is the result of choosing
an incorrect region).
Thanks Chris
